### PR TITLE
[DEV APPROVED] 10126 Copyright notice

### PR DESCRIPTION
**Pull request by margOnline**
[tp 10126](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5682182231901749200&appConfig=eyJhY2lkIjoiRkRBMzk3RTBGNjU3NDg1RDM4QzczMTJFNzI5MDFBN0MifQ==&boardPopup=userstory/10126/silent)

#### Requirement
Add a copyright notice in the footer
- Use translations for the text
- Update the alt tag on the sfgb logo because it was using a non-existent translation key

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by margOnline** [comment]
@MarcoBgn haha, this is why I am not a frontender, looked ok to me. Is this any better?

screen shot 2019-01-10 at 13 35 57

**Comment by MarcoBgn** [comment]
Looks perfect now to me @margOnline! 

Disclaimer: Not a FE dev either so it might still be off 😄

File: app/views/shared/footer/_footer.html.erb [hunk 1]
```diff
@@ -11,6 +11,9 @@
           <%= render 'shared/footer/footer_social' %>
           <%= render 'shared/footer/footer_newsletter' %>
         </div>
+        <div class="footer__copyright">
```
Comment (jiveturkeyJay) [inline app/views/shared/footer/_footer.html.erb]: This should be p tag not a div
you'll probably need to add margin: 0 the the above styles

Comment (margOnline) [reply app/views/shared/footer/_footer.html.erb]: done, @jiveturkeyJay ♻️

File: app/assets/stylesheets/components/_footer.scss [hunk 2]
```diff
@@ -128,3 +128,10 @@
     display: block;
   }
 }
+
+.footer__copyright {
+  margin: 0;
+  color: $color-white;
+  line-height: 0.5rem;
```
Comment (davidtrussler) [inline app/assets/stylesheets/components/_footer.scss]: I think this value should be greater than 1 so the text renders correctly across more than 1 line.

**Pull request merged**